test/e2e/kcm_controller.go

```
@@ -0,0 +1,290 @@
+
+	output, err = oc.WithoutNamespace().Run("describe").Args("quota", "quota43039", "-n", ns).Output()
+	o.Expect(err).NotTo(o.HaveOccurred())
+	o.Expect(output).To(o.ContainSubstring("openshift.io/imagestreams  1     10"))
+
+	err = oc.WithoutNamespace().Run("delete").Args("is", "--all", "-n", ns).Execute()
+	o.Expect(err).NotTo(o.HaveOccurred())
+
+	err = wait.Poll(5*time.Second, 30*time.Second, func() (bool, error) {
+		output, err = oc.WithoutNamespace().Run("get").Args("is", "-n", ns).Output()
+		if err != nil {
+			return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 72) but it returns nil

(nilerr)